### PR TITLE
fix nightly upgrade test

### PR DESCRIPTION
**Pull request by meldafrawi**
Signed-off-by: Mohamed Eldafrawi <mohamed.eldafrawi@rancher.com>

Ref: https://github.com/longhorn/longhorn/issues/2321

### CODE REVIEW

File: manager/integration/tests/common.py [hunk 1]
```diff
@@ -2581,184 +2582,23 @@ def reset_disks_for_all_nodes(client):  # NOQA
...
 
-    default_data_locality_setting = \
-        client.by_id_setting(SETTING_DEFAULT_DATA_LOCALITY)
-    try:
-        client.update(default_data_locality_setting, value="disabled")
-    except Exception as e:
-        print("Exception when update Default Data Locality setting",
-              default_data_locality_setting, e)
+    for setting in client.list_setting():
+        setting_name = setting.definition.name
```
Comment (c3y1huang) [inline manager/integration/tests/common.py:2587]: I think this should be `setting.name`.

**Review by c3y1huang — CHANGES_REQUESTED** [review]

File: manager/integration/tests/common.py (hunk 1 above)
Comment (shuo-wu) [reply manager/integration/tests/common.py:2587]: The setting data is like this:
```
{
  "actions": { },
  "definition": {
    "category": "general",
    "default": "false",
    "description": "By default, Longhorn will block `kubectl drain` action on a node if the node contains the last healthy 
  replica of a volume.\n\nIf this setting is enabled, Longhorn will **not** block `kubectl drain` action on a node even if 
  the node contains the last healthy replica of a volume.",
    "displayName": "Allow Node Drain with the Last Healthy Replica",
    "readOnly": false,
    "required": true,
    "type": "bool",
  },
  "id": "allow-node-drain-with-last-healthy-replica",
  "name": "allow-node-drain-with-last-healthy-replica",
  "timestamp": "2021-03-11T02:44:32.632940164Z",
  "type": "setting",
  "value": "false",
},
```

Comment (meldafrawi) [reply manager/integration/tests/common.py]: Fixed, Thanks

**Review by c3y1huang — APPROVED** [review]
LGTM

**Review by innobead — APPROVED** [review]

**Pull request merged**